compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -597,20 +597,18 @@ createProjectPackageDb opts fps = do
                       createDirectoryIfMissing True workDir
                       -- write the dalf package
                       B.writeFile (workDir </> unitIdStr <.> "dalf") bs
-                      -- we change the working dir so that we get correct file paths for the
-                      -- interface files.
-                      withCurrentDirectory workDir $
-                          generateAndInstallIfaceFiles
-                              dalf
-                              src
-                              opts
-                              dbPathAbs
-                              projectPackageDatabaseAbs
-                              unitIdStr
-                              pkgIdStr
-                              pkgName
-                              mbPkgVersion
-                              deps
+                      generateAndInstallIfaceFiles
+                          dalf
+                          src
+                          opts
+                          workDir
+                          dbPath
+                          projectPackageDatabase
+                          unitIdStr
+                          pkgIdStr
+                          pkgName
+                          mbPkgVersion
+                          deps
 
                       unless (null templInstSrc) $
                           generateAndInstallInstancesPkg
```

Comment:
We do basically the same thing in `generateAndInstallInstancesPkg` so we should probably fix that one as well (fine to leave that for a separate PR).